### PR TITLE
Typesafe is now Lightbend

### PR DESCRIPTION
**Pull request by SethTisue**
review by @adriaanm

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ We are super happy about this, and are eager to make your experience contributin
 
 This is why we're collecting these notes on how to contribute, and we hope you'll share your experience to improve the process for the next contributor! (Feel free to send a PR for this note, send your thoughts to scala-internals, or tweet about it to @adriaanm.)
 
-By the way, the team at Typesafe is: @adriaanm, @lrytz, @retronym, and @SethTisue.
+By the way, the team at Lightbend (formerly Typesafe) is: @adriaanm, @lrytz, @retronym, and @SethTisue.
```
Comment (adriaanm) [inline CONTRIBUTING.md]: and szeiger!

**Comment by adriaanm** [comment]
I haven't confirmed yet we can do a mutable update here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (SethTisue) [inline CONTRIBUTING.md]: revised commit pushed

**Comment by SethTisue** [comment]
integrate-ide failure is unrelated. I reported it on Gitter, scala-ide/scala-ide

File: doc/LICENSE.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Scala is licensed under the [BSD 3-Clause License](http://opensource.org/license
 
 Copyright (c) 2002-2016 EPFL
 
-Copyright (c) 2011-2016 Typesafe, Inc.
+Copyright (c) 2011-2016 Lightbend, Inc.
```
Comment (adriaanm) [inline doc/LICENSE.md]: I would go with `Lightbend, Inc. (Formerly Typesafe, Inc.)` for now.

File: doc/License.rtf [hunk 3]
```diff
@@ -11,7 +11,7 @@
 \fs40 \
 
 \fs26 Copyright (c) 2002-2016 EPFL\
-Copyright (c) 2011-2016 Typesafe, Inc.\
+Copyright (c) 2011-2016 Lightbend, Inc.\
```
Comment (adriaanm) [inline doc/License.rtf]: I would go with `Lightbend, Inc. (Formerly Typesafe, Inc.)` for now.

**Comment by adriaanm** [comment]
I would include the old name in the licenses, but otherwise LGTM

**Comment by SethTisue** [comment]
> I would include the old name in the licenses

done

**Pull request merged**